src/bandwidthRtc.ts

```diff
@@ -91,7 +101,10 @@ class BandwidthRtc {
     const peerConnection = new RTCPeerConnection(RTC_CONFIGURATION);
     this.setupNewPeerConnection(peerConnection, endpointId, mediaTypes);
     mediaStream.getTracks().forEach((track) => {
-      peerConnection.addTrack(track, mediaStream);
+      var sender = peerConnection.addTrack(track, mediaStream);
+      if (track.kind === "audio") {
```

Comment:
should this be a defined constant or is there some existing enum we can use here?  (MediaType?)

Reply:
Good point, I'll check.

Reply:
There isn't one, unfortunately. Can I even define one myself for a field that's internal to WebAudio?

Reply:
I've had to use plain strings in similar situations, not sure why they don't have better defined values/constants.